src/lib/RequestTypes.ts

```diff
@@ -221,6 +250,13 @@ export class AccountsRequest {
                     kind: RequestType.LOGIN,
                     appName: request.appName,
                 } as LoginRequest;
+            case RequestType.LOGOUT:
+            return {
+                kind: RequestType.LOGOUT,
+                appName: request.appName,
+                keyId: request.keyId,
+                keyLabel: request.keyLabel,
+            } as LogoutRequest;
```

Comment:
I guess you can just `return request` here instead of re-constructing it, because there no change from parsed to raw request.